webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -173,9 +199,15 @@ def handle(self, **options):
             if not check_date:
                 raise CommandError("Unrecognized run_time format: %s" % check_date_arg)
         else:
-            check_date = timezone.now() - datetime.timedelta(days=1)
+            check_date = timezone.now()
+
+        # Look at the previous day because we want to look at a whole day.
+        check_date = check_date - datetime.timedelta(days=1)
```

Comment:
This is a little annoying since if your'e running it manually on the command line and give it a date, feels like you expect it to check the crash reports on that date, but instead it does the previous day. However, that's how cronrun is doing it and it's running it most of the time, so I figured I'd cater to that and make a note in the docs.